infrastructure/src/main/kotlin/app/discord/listeners/GuildMemberJoinEventListener.kt

```diff
@@ -12,14 +13,16 @@ class GuildMemberJoinEventListener(
 
     override fun onGuildMemberJoin(event: GuildMemberJoinEvent) {
         this.applicationEventPublisher.publishEvent(
-            this.firstMemberJoin(event = event)
+            this.memberRegister(event = event)
         )
-
     }
 
-    private fun firstMemberJoin(event: GuildMemberJoinEvent) = FirstMemberJoin(
+    private fun memberRegister(event: GuildMemberJoinEvent) = UserRegisterEvent(
```

Comment:
dto로 변환해주는 함수명을 기존에 통일했던 대로 변경해주시기 바랍니다.
```suggestion
    private fun toUserRegisterEvent(event: GuildMemberJoinEvent) = UserRegisterEvent(
```